dev/benchmarking/ddl/2_constraints.sql

```diff
@@ -0,0 +1,153 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+*/
+
+-- BENCHMARK
```

Comment:
Is it typical in postgres to have constraints in a separate file from the tables.  It would be my preference (but we should probably get others opinions) if we were trying to make file sizes reasonable to split by table (and put as many constraints as possible with the table)

Reply:
+1. Happy to do this unless other people feel strongly. It's actually my preference too; it slowed me down to switch between files while double-checking table specs. Will wait a day or two then just do it barring strong counter-opinions.